examples/domains/simple-domain.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: takaro.io/v1
+kind: Domain
+metadata:
+  name: simple-domain
+  namespace: takaro-system
+spec:
+  name: simple-domain
+  settings:
+    maintenanceMode: false
+    publiclyVisible: true
+    allowRegistration: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)